libs/pika/algorithms/tests/performance/benchmark_inplace_merge.cpp

```diff
@@ -236,5 +236,5 @@ int main(int argc, char* argv[])
     PIKA_TEST_EQ_MSG(pika::init(pika_main, argc, argv, init_args), 0,
         "pika main exited with non-zero status");
 
-    return pika::util::report_errors();
+    return 0;
```

Comment:
what happened in case we call `report_errors` twice?

Reply:
The test summary will be printed twice but not much else should happen. Failures will still be reported and importantly the exit code will still be non-zero if tests fail. We have exactly one test where we still call `report_errors` explicitly and I'm not quite sure yet if we can do without it there. _If_ we get rid of all current explicit calls to `report_errors` then we could just hide `report_errors` inside the source file. I'll look into that separately.